<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRoomDialog</class>
 <widget class="QDialog" name="NewRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New room</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Nickname:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nicknameLb">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="settingsBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/settings</normaloff>:/images/settings</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadTableBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Load Table</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/box</normaloff>:/images/box</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../../Engine/EResources.qrc">
        <normaloff>:/images/server</normaloff>:/images/server</iconset>
      </attribute>
      <attribute name="title">
       <string>Host Room</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Room name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="roomNameEdit">
           <property name="text">
            <string>Join my room!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Table Info</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Description:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QPlainTextEdit" name="descriptionTextEdit">
              <property name="plainText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QLineEdit" name="roomTypeEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Type:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Max.:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="maxUserSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="usersLb">
              <property name="text">
               <string>users.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../../Engine/EResources.qrc">
        <normaloff>:/images/wheel</normaloff>:/images/wheel</iconset>
      </attribute>
      <attribute name="title">
       <string>Preferences</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="1" column="3">
          <widget class="QCheckBox" name="portCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Choose best</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Change teams:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QComboBox" name="hideImageComboBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Everyone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nobody</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yours Only</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QComboBox" name="teamSortingComboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>One per team</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Random team</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Same team</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Team sorting:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="portSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maxLength">
            <number>50</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Host Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Hide item:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Add item:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QComboBox" name="addImageComboBox">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Everyone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nobody</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Lock item:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Ban users:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="removeImageComboBox">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Everyone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nobody</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yours Only</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Remove item:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QComboBox" name="banUsersComboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="changeTeamsComboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Everyone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nobody</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="lockImageComboBox">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>Host Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Admins Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Everyone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nobody</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yours Only</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QCheckBox" name="locationCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Display your current location.</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startServerBtn">
       <property name="text">
        <string>Start server</string>
       </property>
       <property name="icon">
        <iconset resource="../../../Engine/EResources.qrc">
         <normaloff>:/images/startServer</normaloff>:/images/startServer</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>roomNameEdit</tabstop>
  <tabstop>roomTypeEdit</tabstop>
  <tabstop>maxUserSpinBox</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>lockImageComboBox</tabstop>
  <tabstop>hideImageComboBox</tabstop>
  <tabstop>addImageComboBox</tabstop>
  <tabstop>removeImageComboBox</tabstop>
  <tabstop>changeTeamsComboBox</tabstop>
  <tabstop>teamSortingComboBox</tabstop>
  <tabstop>banUsersComboBox</tabstop>
  <tabstop>locationCheckBox</tabstop>
  <tabstop>startServerBtn</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>settingsBtn</tabstop>
  <tabstop>loadTableBtn</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Engine/EResources.qrc"/>
 </resources>
 <connections/>
</ui>
